crypto/cipher_extra/test/make_all_legacy_aead_tests.sh

```diff
@@ -10,3 +10,9 @@ go run make_legacy_aead_tests.go -cipher aes256 -mac sha1 -implicit-iv > aes_256
 
 go run make_legacy_aead_tests.go -cipher 3des -mac sha1 > des_ede3_cbc_sha1_tls_tests.txt
 go run make_legacy_aead_tests.go -cipher 3des -mac sha1 -implicit-iv > des_ede3_cbc_sha1_tls_implicit_iv_tests.txt
+
+go run make_legacy_aead_tests.go -cipher aes128 -mac sha1 -stitch > aes_128_cbc_sha1_tls_stitch_tests.txt
```

Comment:
Does the AEAD naming here indicate that the test data is only compatible with `EVP_AEAD` api?
We intend to use `EVP_CIPHER` for the stitched implementation right?

Reply:
> Does the AEAD naming here indicate that the test data is only compatible with EVP_AEAD api?

No, it's not compatible with EVP_AEAD API. The cipher is TLS specific(see below quote). From test data perspective, may I call it legacy AEAD?

https://mta.openssl.org/pipermail/openssl-users/2019-November/011458.html
```
> WARNING: this is not intended for usage outside of TLS and requires
> calling of some undocumented ctrl functions. These ciphers do not
> conform to the EVP AEAD interface."
```

> We intend to use EVP_CIPHER for the stitched implementation right?

Yes, the same APIs from OpenSSL.

Reply:
Think it's fine as-is. Just checking.